tests/test_flows.py

```diff
@@ -814,53 +834,131 @@ def test_client_capability_query(self):
         sta1 = env.Station.create()
         sta2 = env.Station.create()
 
-        association_frame = """00 0e 4d 75 6c 74 69 2d 41 50 2d 32 34 47 2d 31
-01 08 02 04 0b 0c 12 16 18 24 21 02 00 14 30 14
-01 00 00 0f ac 04 01 00 00 0f ac 04 01 00 00 0f
-ac 02 00 00 32 04 30 48 60 6c 3b 10 51 51 53 54
-73 74 75 76 77 78 7c 7d 7e 7f 80 82 3b 16 0c 01
-02 03 04 05 0c 16 17 18 19 1a 1b 1c 1d 1e 1f 20
-21 80 81 82 46 05 70 00 00 00 00 46 05 71 50 50
-00 04 7f 0a 04 00 0a 82 21 40 00 40 80 00 dd 07
-00 50 f2 02 00 01 00 2d 1a 2d 11 03 ff ff 00 00
-00 00 00 00 00 00 00 00 00 00 00 00 00 00 18 e6
-e1 09 00 bf 0c b0 79 d1 33 fa ff 0c 03 fa ff 0c
-03 c7 01 10 """
-
+        association_frame = "00:0e:4d:75:6c:74:69:2d:41:50:2d:32:34:47:2d:31:"\
+                            "01:08:02:04:0b:0c:12:16:18:24:21:02:00:14:30:14:"\
+                            "01:00:00:0f:ac:04:01:00:00:0f:ac:04:01:00:00:0f:"\
+                            "ac:02:00:00:32:04:30:48:60:6c:3b:10:51:51:53:54:"\
+                            "73:74:75:76:77:78:7c:7d:7e:7f:80:82:3b:16:0c:01:"\
+                            "02:03:04:05:0c:16:17:18:19:1a:1b:1c:1d:1e:1f:20:"\
+                            "21:80:81:82:46:05:70:00:00:00:00:46:05:71:50:50:"\
+                            "00:04:7f:0a:04:00:0a:82:21:40:00:40:80:00:dd:07:"\
+                            "00:50:f2:02:00:01:00:2d:1a:2d:11:03:ff:ff:00:00:"\
+                            "00:00:00:00:00:00:00:00:00:00:00:00:00:00:18:e6:"\
+                            "e1:09:00:bf:0c:b0:79:d1:33:fa:ff:0c:03:fa:ff:0c:"\
+                            "03:c7:01:10"
+
+        # first check capability query fails on unconnected station
         debug("Send client capability query for unconnected STA")
-        env.controller.dev_send_1905(env.agents[0].mac, 0x8009,
-                                     tlv(0x90, 0x000C,
-                                         '{} {}'.format(env.agents[0].radios[0].mac, sta1.mac)))
+        mid1 = env.controller.dev_send_1905(env.agents[0].mac, 0x8009,
+                                            tlv(0x90, 0x000C, '{} {}'.format(
+                                                env.agents[0].radios[0].mac, sta1.mac)))
         time.sleep(1)
-        debug("Confirming client capability query has been received on agent")
-        # check that both radio agents received it, in the future we'll add a check to verify which
-        # radio the query was intended for.
-        self.check_log(env.agents[0], r"CLIENT_CAPABILITY_QUERY_MESSAGE")
 
-        debug("Confirming client capability report message has been received on controller")
-        self.check_log(env.controller, r"Received CLIENT_CAPABILITY_REPORT_MESSAGE")
-        self.check_log(env.controller,
-                       r"Result Code= FAILURE, client MAC= {}, BSSID= {}"
-                       .format(sta1.mac, env.agents[0].radios[0].mac))
+        query1 = self.check_cmdu_type_single("client capability query", 0x8009, env.controller.mac,
```

Comment:
Same nitpick about sensless naming of variable.
```suggestion
        client_cap_query = self.check_cmdu_type_single("client capability query", 0x8009, env.controller.mac,
```

Reply:
yeah, I guess I can make this more meaningful, like query_for_error instead of query1 and query_for_success instead of query2.

Reply:
I think "query" would be OK, but as soon as you need more than one, "query1" and "query2" are meaningless.

In this specific case, however, I think it would be better to split the test into two separate tests. The two parts of the test are pretty much independent of each other.

Reply:
I changed the variable names in that test. This time the lines are too long but I think it is OK for test code, what we are testing should be clear in variable names.

Btw I didn't used client_cap_query name since it is evident in the test name and check_cmdu_type error string parameter. I instead changed the name to reflect that we are testing for a query which returns a report with error and then a query with a successful report.